tests/apps/actorfeatures/app.go

```diff
@@ -172,7 +173,12 @@ func appendLog(actorType string, actorID string, action string, start int) {
 }
 
 func getLogs() []actorLogEntry {
-	return actorLogs
+	actorLogsMutex.Lock()
```

Comment:
This was another source of instability in our tests. It was hard to find but it made a difference.